radio/src/translations/de.h

```diff
@@ -584,37 +584,37 @@
 #define TR_MULTI_TELEMETRY             "Telemetry"
 #define TR_MULTI_VIDFREQ               TR(INDENT "Vid. Freq.", INDENT "Video Frequenz")
 #define TR_RF_POWER                    INDENT "RF Power"
-#define TR_MULTI_FIXEDID               TR("FixedID", "Fixed ID")
+#define TR_MULTI_FIXEDID               TR("FesteID", "Feste ID")
 #define TR_MULTI_OPTION                TR(INDENT "Option", INDENT "Optionswert")
-#define TR_MULTI_AUTOBIND              TR(INDENT "Bind Ch.",INDENT "Bind on channel")
-#define TR_DISABLE_CH_MAP              TR("No Ch. map", "Disable Ch. map")
-#define TR_DISABLE_TELEM               TR("No Telem", "Disable Telemetry")
+#define TR_MULTI_AUTOBIND              TR(INDENT "Bind Ka.",INDENT "Bindung an Kanal")
+#define TR_DISABLE_CH_MAP              TR("No Ch. map", "Deaktiviere Ch. map")
+#define TR_DISABLE_TELEM               TR("No Telem", "Deaktiviere Telem.")
 #define TR_MULTI_DSM_AUTODTECT         TR(INDENT "Autodetect", INDENT "Autodetect format")
-#define TR_MULTI_LOWPOWER              TR(INDENT "Low power", INDENT "Low power mode")
+#define TR_MULTI_LOWPOWER              TR(INDENT "Low power", INDENT "reduzierte Leistung")
 #define TR_MULTI_LNA_DISABLE           INDENT "LNA disable"
 #define TR_MODULE_TELEMETRY            TR(INDENT "S.Port", INDENT "S.Port link")
 #define TR_MODULE_TELEM_ON             TR("EIN", "Aktiviert")
-#define TR_DISABLE_INTERNAL            TR("Disable int. RF", "Disable internal RF")
+#define TR_DISABLE_INTERNAL            TR("Deaktiv. int. RF", "Deaktiviere int. RF")
 #define TR_MODULE_NO_SERIAL_MODE       TR("!serial mode", "Not in serial mode")
 #define TR_MODULE_NO_INPUT             TR("No input", "No serial input")
-#define TR_MODULE_NO_TELEMETRY         TR3("Keine Telem.", "Keine MULTI_TELEMETRY", "Keine Telemetrie (aktiviere MULTI_TELEMETRY)")
-#define TR_MODULE_WAITFORBIND          "Bind to load protocol"
-#define TR_MODULE_BINDING              "Binding"
-#define TR_MODULE_UPGRADE_ALERT        TR3("Upg. needed", "Module upgrade required", "Modul\nUpdate nötig")
-#define TR_MODULE_UPGRADE              TR("Upg. advised", "Modulupdate empfohlen")
-#define TR_REBIND                      "Rebinding required"
+#define TR_MODULE_NO_TELEMETRY         TR3("Keine Telem.", "Keine MULTI_TELEMETRIE", "Keine Telemetrie (aktiviere MULTI_TELEMETRIE)")
+#define TR_MODULE_WAITFORBIND          "Warten auf Bindung"
+#define TR_MODULE_BINDING              "Bindung"
+#define TR_MODULE_UPGRADE_ALERT        TR3("Upg. nötig", "Modul Upgrade nötig", "Modul\nUpgrade nötig")
+#define TR_MODULE_UPGRADE              TR("Upg. empf.", "Modul Upgrade empf.")
+#define TR_REBIND                      "Neu binden nötig"
 #define TR_REG_OK                      "Registration ok"
-#define TR_BIND_OK                     "Bind successful"
+#define TR_BIND_OK                     "Bindung erfolgreich"
```

Comment:
Same here
"Binden erfolgreich"
because
"Bindung" is a connection and "Binden" ist the act of connecting

Reply:
done